### ISSUE TITLE
The outdated/obsolete/discarded melpa repository URL.

### ISSUE THREAD

**Issue by hongyi-zhao** [jwiegley/use-package#933]
The following descriptions of use-package URL in the melpa repository in the github repo of this project are outdated:
```
$ egrep -inR 'melpa.milkbox.net' .
./README.md:5:[![MELPA](http://melpa.milkbox.net/packages/use-package-badge.svg)](http://melpa.milkbox.net/#/use-package)
./README.md:65:[MELPA](http://melpa.milkbox.net/) (recommended).
```
Currently, the actively maintained melpa repository is `https://melpa.org`.

Regards,
HY

**Comment by nopdotcom** [comment]
This has been fixed as of e2d173b1200865a9efd5c2066831a230497582c0 so this can be closed.

**Comment by skangas** [comment]
Thanks @nopdotcom, I'm therefore closing this issue.
